app/api/routes.py

```diff
@@ -688,7 +688,7 @@
     TicketDetail,
     'ticket_detail',
     '/tickets/<int:id>',
-    '/attendees/<int:attendee_id>/ticket',
+    '/attendees/<int:attendee_id>/ticket'
```

Comment:
Same as previous PR, please keep final `,` -- out of convenience as well as minimizing diffs!